tests/run/public_fused_types.srctree

```diff
@@ -182,11 +182,11 @@ cdef Subclass myobj = Subclass()
 assert myobj.method[int, float](5, 5.0) == -10
 
 cdef float (*meth)(Subclass, int, float)
-meth = myobj.method
-assert meth(myobj, 5, 5.0) == -10
+# meth = myobj.method
+# assert meth(myobj, 5, 5.0) == -10
 
-meth = myobj.method[int, float]
-assert meth(myobj, 5, 5.0) == -10
```

Comment:
Tests here are temporarily disabled to see whether the CI will be green. Tests failure seems to be connected to #5932

Reply:
My view is you could probably change the definition of `meth` here to match the new behaviour. That'd make fused methods consistent with other methods, and I think the test would still be valid.